routes/web.public.php

```diff
@@ -30,4 +20,9 @@
 Route::get('updater/update', [UpdateController::class, 'update']);
 Route::get('updater/version', [UpdateController::class, 'version']);
 
-Route::post('api/contact-form', 'ContactFormController@send')->middleware('cors');
+// iDeal integration
+Route::post('iDeal-webhook', [iDealController::class, 'webhook']);
```

Comment:
This should be in the API endpoint

Reply:
Somehow this causes the corresponding `action()` to not be defined anymore... not sure why yet

Reply:
never mind, forgot importing the controller